bin/minifollowups/pycbc_injection_minifollowup

```diff
@@ -305,19 +306,31 @@ for num_event in range(num_events):
                             use_exact_inj_params=True)
 
     for curr_ifo in args.single_detector_triggers:
+        # Finding loudest template in this detector near to the injection:
```

Comment:
The changes here were possible before, but this is just to implement the select() function and make things quite a bit faster